tls-attestation/draft-fossati-tls-attestation.md

```diff
@@ -550,11 +589,124 @@ the ClientHello.
                         |<---------------------->|
                         |                        |
 ~~~~
-{: #figure-overview title="Cloud Confidential Computing Example Exchange."}
+{: #figure-cc-example title="Example Exchange with Server as Attester."}
 
 ## IoT Device Onboarding
 
-TBD.
+In this example, an IoT is onboarded to a cloud service provider (or to a
+network operator). In this scenario there is typically no a-priori
+relationship between the device and the cloud service provider that 
+will remote manage the device.
+
+In such scenario, the cloud service provider wants to make sure
+that the device runs the correct version of firmware, has not been 
+rooted, is emulated, or cloned.
+
+The protocol flow is shown in {{figure-cc-example}} where the client 
+is the attester while the server is the relying party.
+
+The flow starts with the client initiating a TLS exchange with the TLS
+server operated by the cloud service provider. The client indicates
+what evidence types it supports.
+
+The server obtains a nonce from the verifier, in real-time or from a
+reserved nonce range, and returns it to the client alongside the
+selected evidence type. Since the evidence will be returned in the
+Certificate message the server has to request mutual authentication
+via the CertificateRequest message.
+
+The client, when receiving the EncryptedExtension with the 
+evidence_proposal, will proceed by invoking a local API to
+request the attestation.  The returned evidence binds the identity key
+with the workload and platform identity and security state.  The client
```

Comment:
not sure in this case it is correct to qualify it as a workload like in the CC case.  I'd just say "identity key with platform identity and security state"